viadot/sources/outlook.py

```diff
@@ -182,10 +182,13 @@ def _get_messages_from_mailbox(
                     recivers_list = fetched.get("toRecipients")
                     recivers = " "
                     if recivers_list is not None:
-                        recivers = ", ".join(
-                            reciver["emailAddress"]["address"]
-                            for reciver in recivers_list
-                        )
+                        for reciver in recivers_list:
+                            add_string = f", {reciver['emailAddress']['address']}"
+                            if sum(list(map(len, [recivers, add_string]))) >= 8000:
```

Comment:
Why do you assign `8000`?

Reply:
8000 is the maximum to no overload data in a `varchar()` variable. More than that data is right truncated and the entire row is useless.